requirements.txt

```diff
@@ -1,2 +1,2 @@
-requests>=2.20
-flake8>=3.8,<=4.0
+requests>=2.20,<3.0
```

Comment:
@funilrys since you are more familiar with Python, wouldn't this also work? It seems smaller and matches what people use on npm.

```
requests ~=2.20
flake8 ~=4.0
```

Not sure if it makes sense in the pip world, though.

Reply:
Hey @XhmikosR, yes that will work. It will restrict to the major version.

Reply:
Thanks! I'm gonna switch to that and we can revisit if needed.